html/supertokens_python/constants.html

```diff
@@ -43,7 +43,7 @@ <h1 class="title">Module <code>supertokens_python.constants</code></h1>
 from __future__ import annotations
 
 SUPPORTED_CDI_VERSIONS = [&#34;5.2&#34;]
-VERSION = &#34;0.28.0&#34;
+VERSION = &#34;0.28.1&#34;
```

Comment:
This is inconsistent with the changelog

Reply:
Apologies, these seem to be an artifact of using `0.29` as a base branch (since we considered releasing this with other changes), a few commits were added to `0.28` and not on `0.29` yet.
Both of these version issues should be fixed now. Changed the base branch to `0.28`.